src/views/InteractiveMap/Card.vue

```diff
@@ -261,7 +261,11 @@
       </v-container>
 
       <!-- ecModels visualizing enzyme usage -->
-      <v-container v-if="showEnzymeUsageSlider" fluid class="pa-0">
+      <v-container
+        v-if="showEnzymeUsageSlider && model && model.ec_model"
```

Comment:
I think it makes more sense to include this logic where `showEnzymeUsageSlider` is defined (L521 in `Card.vue`):

```js
    showEnzymeUsageSlider() {
      return (
        this.card.type == "DataDriven" &&
        this.card.sample &&
        this.card.sample.proteomics.length > 0
      );
    },
```